PipelinesTasks/PRMetrics/buildAndReleaseTask/Updaters/CodeMetricsHelpers.ts

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable indent */
+/* eslint-disable semi */
+/* eslint-disable space-before-function-paren */
+export function isNullOrWhitespace(input: string | null) {
```

Comment:
Thanks for converting this. However, I don't think we really require this – I was actually just using it because it is built into PowerShell.

I'd suggest just do the "if (input)" check instead, which provides most of the functionality in the JavaScript preferred way